LiskHQ/lisk-nano
# Add error messages for autocomplete - Closes #823

What's the Problem?
We make an unnecessary request every 10 second just to check if the client is online or not.
[#823](https://github.com/LiskHQ/lisk-nano/issues/823)

What did you do?
- added an extra check and added error message
- added one translation

How can I test this?
- go to "Vote" 
- check for both "Add vote to" and "Remove vote from" that : 
   - the search is still working (showing results) 
   - when you search for something that doesn't exist, the autocomplete should be blank and the 
     error message "No matches" appears

It should look like this:
<img width="838" alt="screen shot 2017-10-05 at 11 46 14" src="https://user-images.githubusercontent.com/9592216/31221092-dc052922-a9c2-11e7-94bc-314512d485b7.png">
